_data/Farid-Wakim/Farid-Wakim.md

```diff
@@ -0,0 +1,7 @@
+---
+name: FARID-WAKIM # No longer than 28 characters
+institution: LEBANESE-INTERNATIONAL-UNIVERSITY 🚩 # no longer than 58 characters
+profile_pic: farid.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Incorrect file name, please do one of the following:
1. Change `farid.png` to `Farid-Wakim.jpg` in the markdown file.
2. Rename `Farid-Wakim.jpg` to `farid.png`.
I'd suggest you make the change in the markdown file!